lib/generated/intl/messages_pl.dart

```diff
@@ -22,22 +22,38 @@ class MessageLookup extends MessageLookupByLibrary {
   final messages = _notInlinedMessages(_notInlinedMessages);
   static _notInlinedMessages(_) => <String, Function> {
     "aboutApp" : MessageLookupByLibrary.simpleMessage("O aplikacji"),
-    "apiConnectionException" : MessageLookupByLibrary.simpleMessage("Brak połączenia sieciowego"),
     "apiInvalidUrlException" : MessageLookupByLibrary.simpleMessage("Nie znaleziono danych"),
     "apiServerException" : MessageLookupByLibrary.simpleMessage("Błąd na serwerze"),
     "apiUnauthorizedException" : MessageLookupByLibrary.simpleMessage("Błąd autoryzacyjny"),
     "apiUnknownException" : MessageLookupByLibrary.simpleMessage("Nieznany błąd"),
     "appInfoFeatureDescription" : MessageLookupByLibrary.simpleMessage("Najnowsze informacje z całego świata"),
     "appInfoTechnicalDescription" : MessageLookupByLibrary.simpleMessage("Aplikacja w całości napisana we Flutterze"),
+    "createAccount" : MessageLookupByLibrary.simpleMessage("Utwórz konto"),
     "darkMode" : MessageLookupByLibrary.simpleMessage("Tryb ciemny"),
+    "email" : MessageLookupByLibrary.simpleMessage("E-mail"),
     "emailed" : MessageLookupByLibrary.simpleMessage("Wysyłane e-mailem"),
+    "emptyFieldError" : MessageLookupByLibrary.simpleMessage("To pole nie może być puste"),
+    "firebaseInitializationError" : MessageLookupByLibrary.simpleMessage("Nieudana inicjalizacja firebase, zrestartuj aplikację"),
```

Comment:
mało user friendly - `Błąd inicjalizacji. Zrestartuj aplikację` - może nie jest to copywriter lvl master, ale nie ma w komunikacie firebase :D